### DISCUSSION TITLE
Video format (4:3, 16:9 etc)olution" parameter in the Con

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#419] [Q&A]
Finally got my video stream to display in VLC and have the redimentary remote working.

I've changed the "Resolution" parameter in the config.ini to 16  which the read.md file says is 1920x1080. 
When I look at the "current media information" in VLC ( Tools - Media inormation) it has codec H264, and video resolution 640x480

![image](https://user-images.githubusercontent.com/117101460/205637032-6f64dfed-9d11-43a5-b4ad-6319bb85b7c4.png)

Can anyone tell me how to get a 16:9 picture shape. In Slingplayer 2.0.4 I would have gone for anamorphic shape for the video.

**Comment by GerryDazoo** [comment]
Click on video menu then aspect ratio. My github is turning into VLC support! 
The slingbox will not up convert the incoming video stream so I suspect your feeding the box 640x480 OR your using a box that doesn't support that resolution. (Solo?)

**Comment by DoogieHowitzerWMD** [comment]
I see you have "ProHD" in your url for the stream, so assuming

- you have a ProHD
- you have it hooked up to component as the video source
- you have video source set to "3" in the config file
- your cable box doesn't have a "video format override" parameter that is set to SD (one of mine had this enabled for an old CRT tv, but that cable box already downscales the output to SD specifically for RF connections anyway)
- your server logs show the correct config file is being specified when the server initially locates the slingbox

then we can probably say the issue is coming from VLC.

**Comment by gbjbasdw** [comment]
I think the issue is as Gerry suggests - the HDMI to Composite converter box that feeds video into the Slingbox itself.
My Tivi only has HDMI output but my Slingbox ProHD input only has composite/component and Video-S.

**Reply by sp222222** [reply]
yeah my ProHD on tuner gives me 1280x544 on the local news channel. I use a HDMI to component converter box but this one (newer) does not have a button to change/toggle the output. I'm pretty sure my config is 16.

**Reply by Toei79** [reply]
i though component gives you 1280 its what i get with directv component. 

then by coincidence i get one lg dvd recorder in thrift store , with hdmi and component but via component you can get only 480p. nothing more.

**Reply by bradut** [reply]
I have also noticed the difference between image resolution provided by various TV boxes:
- my home TV box gives me only 480p on both component and antenna/coax (that's OK, is just my testing environment)
- at a remote location the antenna coax provides a much higher resolution [cannot give exact values yet].
